doc/input-output-reference/src/overview/group-surface-construction-elements.tex

```diff
@@ -240,168 +240,98 @@ \subsubsection{Inputs}\label{inputs-3-036}
 
-\paragraph{Field: Deep Layer Penetration
-	Depth}\label{field-deep-layer-penetration-depth}
+\paragraph{Field: Deep Layer Penetration Depth}\label{field-deep-layer-penetration-depth}
 
-The Deep Layer Penetration Depth is the fictitious thickness of the deep
-layer in meters, and is used to calculate the volume of material that
-participates in long-term moisture transfer and storage. This layer has
-a uniform moisture content, and can be considered a
-\emph{lumped-capacitance}. The deep penetration depth is based on the
-amount of material that interacts with the surface layer when subject to
-cyclic relative humidity variation. The deep penetration depth can be
-estimated with the following equation:
+The Deep Layer Penetration Depth is the fictitious thickness of the deep layer in meters, and is used to calculate the volume of material that participates in long-term moisture transfer and storage. This layer has a uniform moisture content, and can be considered a \emph{lumped-capacitance}. The deep penetration depth is based on the amount of material that interacts with the surface layer when subject to cyclic relative humidity variation. The deep penetration depth can be estimated with the following equation:
 
 \[ d_{EMPD,deep} = \sqrt{\frac{\delta_{perm} P_{sat} \tau_{deep}} {\rho_{material} \frac{du}{d\phi} \pi}} \]
 
-where each term is the same as the surface layer, except that the cycle
-period is different. This is usually on the order of weeks for the deep
-layer.
-
-If this field is left blank or set to \texttt{autocalculate},
-the above equation will be used to calculate the deep layer penetration depth
-assuming a $\tau_{deep}$ of three weeks. To use a period different than 3 weeks, the equation above can be used to calculate the penetration depth based on a different value of $\tau_{deep}$. The penetration depth can also be entered as an empirical value, as in \href{http://dx.doi.org/10.1016/j.enbuild.2016.02.008}{\emph{Woods and Winkler, 2016}}.
+where each term is the same as the surface layer, except that the cycle period is different. This is usually on the order of weeks for the deep layer.
 
+If this field is left blank or set to \texttt{autocalculate}, the above equation will be used to calculate the deep layer penetration depth assuming a $\tau_{deep}$ of three weeks. To use a period different than 3 weeks, the equation above can be used to calculate the penetration depth based on a different value of $\tau_{deep}$. The penetration depth can also be entered as an empirical value, as in \href{http://dx.doi.org/10.1016/j.enbuild.2016.02.008}{\emph{Woods and Winkler, 2016}}.
 
```

Comment:
Some weird line-broken issues, happening a lot in a few particular sections across the documentation.